num_list.py

```diff
@@ -19,5 +21,8 @@ def print_each_number_and_its_square():
     # The square of 10 is 100
     # ...etc
     # TODO - Write your code below this line.  (Keep the indentation)
-    
+
+
+for i in nums:
```

Comment:
Check your indentation on this part of your code, could be what is causing the failed tests.